code/game/objects/items/weapons/twohanded.dm

```diff
@@ -511,4 +513,61 @@
 	return 0
 
 /obj/item/weapon/twohanded/vibro_weapon/update_icon()
-	icon_state = "hfrequency[wielded]"
\ No newline at end of file
+	icon_state = "hfrequency[wielded]"
+
+/*
+ * Bone Axe
+ */
+/obj/item/weapon/twohanded/fireaxe/boneaxe  // Blatant imitation of the fireaxe, but made out of bone.
+	icon_state = "bone_axe0"
+	name = "bone axe"
+	desc = "A large, vicious axe crafted out of several sharpened bone plates and crudely tied together. Made of monsters, by killing monsters, for killing monsters."
+
+/obj/item/weapon/twohanded/fireaxe/boneaxe/update_icon()  //Currently only here to fuck with the on-mob icons.
+	icon_state = "bone_axe[wielded]"
+	return
+
+/obj/item/weapon/twohanded/fireaxe/boneaxe/suicide_act(mob/user)
```

Comment:
uhhh not unless you make it inherit the whole proc your overriding the return. 

if your not changing the text just let it inheret its suicide act from its parent.